ideconfigs/intellij-complex-nextraq-style.xml.xml

```diff
@@ -0,0 +1,608 @@
+<code_scheme name="NexTraqStyle" version="173">
```

Comment:
![image](https://user-images.githubusercontent.com/61692292/234570201-eeab0f09-7120-4ed1-a868-955c9f15e434.png)

this indentation is not working correctly, one tab instead:

![image](https://user-images.githubusercontent.com/61692292/234570299-9336f12e-4afd-4fc5-b4d9-f963f752f337.png)

Reply:
Fixed